content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -303,6 +317,11 @@ for x, y in list:
 
 </LanguageSection>
 
+<Warning>
+In order to sort arrays in a descending order, or in any custom order, you'll need to define your own **custom comparator**. This is covered in
```

Comment:
i mean you don't necessarily have to, just that it might not be the most efficient

Reply:
what's that supposed to mean

Reply:
you can multiply all the elements by -1 and sort them normally, which will give you the reverse order. not very efficient but it's technically another way to do it

Reply:
Also not sure if a warning block is the best here, maybe an info block is better

Reply:
^ info block :+1: